Rakefile

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rspec/core/rake_task'
+RSpec::Core::RakeTask.new(:spec)
```

Comment:
This is neat - haven't seen it before.
My understanding from skimming docs is that it allows you to test exit status for a rake task within a spec - is that correct?

Docs also recommends wrapping that in a `rescue` clause, in case rspec isn't available (on prod).